src/vision/status_quo/barbara_trims_a_stacktrace.md

```diff
@@ -0,0 +1,125 @@
+   9: tokio::park::thread::CachedParkThread::block_on::{{closure}}
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/park/thread.rs:263:54
+  10: tokio::coop::with_budget::{{closure}}
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/coop.rs:106:9
+  11: std::thread::local::LocalKey<T>::try_with
+             at /home/serg/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/thread/local.rs:272:16
+  12: std::thread::local::LocalKey<T>::with
+             at /home/serg/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/thread/local.rs:248:9
+  13: tokio::coop::with_budget
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/coop.rs:99:5
+  14: tokio::coop::budget
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/coop.rs:76:5
+  15: tokio::park::thread::CachedParkThread::block_on
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/park/thread.rs:263:31
+  16: tokio::runtime::enter::Enter::block_on
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/runtime/enter.rs:151:13
+  17: tokio::runtime::thread_pool::ThreadPool::block_on
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/runtime/thread_pool/mod.rs:71:9
+  18: tokio::runtime::Runtime::block_on
+             at /home/serg/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.3.0/src/runtime/mod.rs:452:43
+  19: stacks_rs::main
+             at ./src/main.rs:1:1
+  20: core::ops::function::FnOnce::call_once
+             at /home/serg/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
+note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
+```
+
+Barbara finds the text overwhelming. She can't just browse it to figure out what code is affected. Instead, she pops up a new tab with gist.github.com copies the text into that handy text box and starts deleting stuff. To start, she deletes the first few lines until her code appears, then she deletes:
+
+* the extra lines from calls to `poll` that are introduced by the async fn machinery;
+* the bits of code that come from tokio that don't affect her;
+* the intermediate wrappers from the standard library pertaining to thread-local variables.
+
+She's a bit confused by the `::{closure}` lines on her symbols but she learned by now that this is normal for `async fn`. After some work, she has reduced her stack to this:
+
+```
+thread 'main' panicked at 'something bad happened here', src/main.rs:16:5
+stack backtrace:
+   1: stacks_rs::process_one::{{closure}} at ./src/main.rs:16:5
```

Comment:
nit: maybe the top level module should be `slow` instead of `stacks_rs`. Not too important thought 😄

Reply:
I wondered if anyone would notice that :)